public/index.html

```diff
@@ -11,7 +11,7 @@
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
     <link rel="preconnect" href="https://fonts.googleapis.com">
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>  
-    <link href="https://fonts.googleapis.com/css2?family=Alegreya+Sans+SC&family=Architects+Daughter&family=Bitter&family=Gloria+Hallelujah&family=Montserrat&family=Permanent+Marker&family=Quicksand:wght@300&family=Shadows+Into+Light&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Alegreya+Sans+SC&family=Architects+Daughter&family=Bitter&family=Gloria+Hallelujah&family=Montserrat&family=Permanent+Marker&family=Quicksand&family=Shadows+Into+Light&display=swap" rel="stylesheet">
```

Comment:
It is good to get rid of the font weight in the index file so that we are able to change it elsewhere in the project.